src/main/java/net/openhft/chronicle/threads/EventGroup.java

```diff
@@ -352,4 +352,11 @@ protected void performClose() {
     public boolean runsInsideCoreLoop() {
         return core.runsInsideCoreLoop();
     }
+
+    @Override
+    protected boolean isRunningOnThread(Thread currentThread) {
+        return core != null && core.thread() == currentThread ||
```

Comment:
can de-dupe or use `net.openhft.chronicle.core.threads.EventLoop#runsInsideCoreLoop`

Reply:
I have used `runsInsideCoreLoop()` in here